guide/daily-spending-wallet/requesting.md

```diff
@@ -34,440 +34,400 @@ Illustration source
 
-{% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/funding/opening-channel.png"
-   retina = "/assets/images/guide/daily-spending-wallet/funding/opening-channel@2x.png"
-   layout = "shadow"
-   caption = "Using Lightning services, a channel can be opened on-demand if there is no inbound liquidity when a payment is received."
-   alt-text = "Screen showing channel being opened."
-   width = 250
-   height = 541
-%}
-
-{% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/funding/channel-opened.png"
-   retina = "/assets/images/guide/daily-spending-wallet/funding/channel-opened@2x.png"
-   layout = "shadow"
-   caption = "Indicate to users that, if they have no inbound liquidity, they should get some before requesting a payment."
-   alt-text = "Screen showing that inbound liquidity is required."
-   width = 250
-   height = 541
-%}
+When denominating invoices in fiat, custom expiries should be used to prevent exposure to price volatility.
 
-{% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/funding/no-inbound.png"
-   retina = "/assets/images/guide/daily-spending-wallet/funding/no-inbound@2x.png"
-   layout = "shadow"
-   caption = "Guide users as to how to obtain inbound liquidity so they can receive payments."
-   alt-text = "Screen showing options to receive inbound liquidity."
-   width = 250
-   height = 541
-%}
+If requesting a specific amount denominated in fiat, use a shorter expiry and have the invoice refresh each time it expires. For a wallet designed for in-person payments, a 30 - 60 second expiry works well.
 
-</div>
+If there is no amount defined on the invoice, the invoice is denominated in bitcoin, or the invoice needs to be shared in a message, then use a longer expiry, like 24 hours.
 
-### On-chain
+While it is good to allow users to define their own custom expiries, remember that only advanced users will do this. Choose a sensible default that makes sense for your user-base and how the wallet is intended to be used.
 
-Receiving on-chain involves the sender broadcasting a transaction to the network for it to be confirmed.
+For some background on situations the user might encounter with different invoice expiries, see this [blog post](https://d.elor.me/2022/01/lightning-invoice-expiration-ux-considerations/) from designer Stephen DeLorme.
 
-<div class="center" markdown="1">
+<div class="image-slide-gallery">
 
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/requesting/confirming.png"
-   retina = "/assets/images/guide/daily-spending-wallet/requesting/confirming@2x.png"
-   layout = "float-left-desktop -background -shadow"
-   caption = "For on-chain, keep users updated on incoming pending transactions."
-   alt-text = "Screen showing on-chain transaction being confirmed."
+   image = "/assets/images/guide/daily-spending-wallet/requesting/Fees@2x.png"
```

Comment:
The title and paragraph are about invoice expirations, but this screen is about fees? Remove it?

Reply:
That was showing the wrong image, was suppose to be the settings. Bout to push update